honeybee_energy/hvac/allair/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Template All-air HVAC definitions.
+
+All-air systems provide both ventilation and satisfaction of heating + cooling
+demand with the same stream of warm/cool air. As such, they often have really
+good humidity control. However, because such systems often involve the 
+
+Properties:
+    * HVAC_TYPES_DICT: A dictionary containing pointers to the classes of each
+        HVAC system. The keys of this dictionary are the names of the HVAC
+        classes (eg. 'VAV').
+    * EQUIPMENT_TYPES_DICT: A dictionary containing pointers to the classes of
+        the HVAC systems. The keys of this dictionary are the names of the HVAC
+        systems as they appear in the OpenStudio standards gem and include the
+        specific equipment in the system (eg. 'VAV chiller with gas boiler reheat').
```

Comment:
I think the term "EQUIPMENT_TYPE" in general might be confusing, as the description here states: it is really a type of systems. Equipment in OpenStudio is more like a part of the system.

it might be better to change it to "SYSTEM_TYPE" or "SYSTEM_TEMPLATE" in general

Reply:
I originally called it `system_type` but this was just too similar to the `type` key that all of the schemas have and that refers to the class of the system.  So, if you ask me, "VAV" sounds like a system type but "VAV chiller with gas boiler reheat" sounds like a specification of the equipment within the system. I think it's just better to make the terms different from one another to avoid confusion. We can revisit this again in the honeybee-schema PR if Mostapha feels similarly.